### PR TITLE
Le Chat Bot

### PR DESCRIPTION
**Pull request by smirrebinx**
URL to deployed app:
https://melodic-stardust-81c7d1.netlify.app/

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,51 +1,196 @@
...
+  showMessage(
+    "Welcome to Cat Bot the Chat bot. What's your first name?",
+    "bot"
+  );
+  chat.scrollTop = chat.scrollHeight;
+};
+
+const favoritePersonality = () => {
+  showMessage(
+    `Hello ${userName}! Which cat personality is your favorite?`,
```
Comment (EmiliaGA) [inline code/script.js:56]: Love the idea and styling of your page! Over all a great job 👍 You seem to be following the general requirements and more and your code is simple to read. The only thing missing for me is some comments to make it more clear wich function does what, but I think it is quite self explanatory anyways, so not necessarily anything you have to change.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 
 ## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+To approach the problem a demo app from Technigo and previous students were used as inspiration. To understand how the chat bot could work other students' code was first implementet and then refactored with additional functions and eventlisteners. Information and code snippets from various blogs and forums were also used and some guidance was needed from a more experienced coder to solve some of the more complicated functions. Future iterations of the chat bot will include a more user friendly experience for a mobile user when it comes to scrolling. 
 
 ## View it live
-
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://melodic-stardust-81c7d1.netlify.app/
```
Comment (EmiliaGA) [inline README.md:11]: The chatbot is working really good and has a great idea to it! 🥇

**Review by EmiliaGA — APPROVED** [review]
Keep up the good job!

File: code/script.js (hunk 1 above)
Comment (smirrebinx) [reply code/script.js:56]: Thank you! Good suggestion with comments on the code explaining the functions.

**Pull request closed without merging**